src/Entity/KeysVision.php

```diff
@@ -82,58 +79,63 @@ class KeysVision
      * @Assert\Length(
      *      min = 2,
      *      max = 50,
-     *      minMessage = "Votre texte doit faire au moins {{ limit }} charactères",
-     *      maxMessage = "Votre texte ne peut pas faire plus de {{ limit }} charactères"
+     *      minMessage = "Votre texte doit faire au moins {{ limit }} caractères",
+     *      maxMessage = "Votre texte ne peut pas faire plus de {{ limit }} caractères"
      * )
      */
     private string $text4;
 
     /**
      * @ORM\Column(type="string", length=50, nullable=true)
+     * @Assert\NotBlank(message="Ce champ ne peut pas être vide")
      * @Assert\Length(
-     *      min = 2,
```

Comment:
Why here is min=1 and max=5, and before is min=2 and max=50.
And it's write length=50.